### PR TITLE
feat: more alarm data and code cleanup

### PR DESCRIPTION
**Pull request by chemelli74**

### CODE REVIEW

File: src/aiocomelit/api.py [hunk 1]
```diff
@@ -308,59 +377,81 @@ async def set_zone_status(self, index: int, action: str) -> bool:
...
-            vedo = await self.get_zone_status(i)
-            alarms.update({i: vedo})
+            area = await self._create_area_object(
+                reply_json_area_desc, reply_json_area_stat, i
+            )
+            areas.update({i: area})
+
+        list_zones: list[int] = reply_json_zone_desc["in_area"]
+        zones = {}
+        for i in range(len(list_zones)):
```
Comment (bdraco) [inline src/aiocomelit/api.py:448]: Maybe use enumerate here

File: src/aiocomelit/api.py [hunk 2]
```diff
@@ -286,18 +300,73 @@ class ComelitVedoApi(ComelitCommonApi):
...
+
+    async def _translate_area_status(
+        self, area: ComelitVedoAreaObject
+    ) -> AlarmAreaState:
+        """Translate AREA status."""
 
-        for field in ALARM_FIELDS.keys():
-            if getattr(zone, field):
-                return ALARM_FIELDS[field]
+        for field in ALARM_AREA_STATUS.keys():
```
Comment (bdraco) [inline src/aiocomelit/api.py:320]: Keys is default and not needed

File: src/aiocomelit/api.py [hunk 3]
```diff
@@ -286,18 +300,73 @@ class ComelitVedoApi(ComelitCommonApi):
...
-        self._json_desc: dict[str, Any] = {}
 
-    async def _translate_zone_status(self, zone: ComelitVedoObject) -> str:
-        """Translate Zone status."""
+    async def _translate_zone_status(
+        self, zone: ComelitVedoZoneObject
+    ) -> AlarmZoneState:
+        """Translate ZONE status."""
+
+        for status in ALARM_ZONE_STATUS.keys():
```
Comment (bdraco) [inline src/aiocomelit/api.py:309]: Keys is default

File: src/aiocomelit/api.py (hunk 1 above)
Comment (chemelli74) [reply src/aiocomelit/api.py:448]: But I need a index later on:

```python
        for i in range(len(list_zones)):
            if not list_zones[i]:
```

**Pull request merged**